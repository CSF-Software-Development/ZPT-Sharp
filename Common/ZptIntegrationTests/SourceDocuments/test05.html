<html>

  <body metal:define-macro="body">

    <h1>This is the body of test5</h1>

  </body>

</html>
