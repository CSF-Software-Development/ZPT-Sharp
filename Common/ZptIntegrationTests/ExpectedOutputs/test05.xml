<?xml version="1.0" encoding="utf-8"?>
<html>
	<body>
		<h1>This is the body of test5</h1>
	</body>
</html>